aspnetcore/fundamentals/minimal-apis/7.0-samples/WebMinOpenApi/projectFile.xml

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>    
     <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.*-*" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
```

Comment:
This will be out of date soon. Can't we a [Floating Versions](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#floating-versions)
`Version="6.*-*" />`

Reply:
I'm a little hesitant to use floating versions for external dependencies. I'd rather pin to versions that we've tested against. This matches the version we are shipping with in the templates for 7.0...